10.9-libcxx/stable/main/finkinfo/x11/xft2-dev.info

```diff
@@ -66,8 +62,8 @@ SplitOff: <<
 		x11-shlibs
 	<<
 	Files: lib/xft2/lib/libXft.2.*dylib
-	Shlibs: %p/lib/xft2/lib/libXft.2.dylib 5.0.0 %n (>= 2.2.0-1)
-	DocFiles: AUTHORS COPYING ChangeLog NEWS README
+	Shlibs: %p/lib/xft2/lib/libXft.2.dylib 6.0.0 %n (>= 2.2.0-1)
```

Comment:
The parenthetical in the Shlibs field is a package-version string that indicates when the compatibility_version value changed. "As of fink package 2.3.8-1, it jumped from 5.0.0 to 6.0.0", so it should be "(>= 2.2.0-1)" not "(>= 2.2.0-1)". The Shlibs tag is indeed confusing to lots of people!